projects/lib/src/timecontrol.cpp

```diff
@@ -321,7 +321,7 @@ void TimeControl::startTimer()
 	m_time.start();
 }
 
-void TimeControl::update()
+void TimeControl::update(bool apply_increment)
```

Comment:
Can you use our naming convention for variables (`m_memberVariable` for class members variables and `normalVariable` for others)? Then this parameter would be `applyIncrement`, `new_time_left` would be `newTimeLeft` etc.